packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -34,6 +34,20 @@ import AddressParser from '../models/address-parser'
 import MultisigConfigModel from '../models/multisig-config'
 import { generateRPC } from '../utils/ckb-rpc'
 
```

Comment:
We need to add some comments explains why the `compressPublicKey()` is needed when importing hardware wallets.